docs/basic/merge-sort.md

```diff
@@ -107,17 +107,11 @@ def merge_sort(ll, rr):
 
 Merge sort can be used to find the [inversion number](https://en.wikipedia.org/wiki/Inversion_(discrete_mathematics)#Inversion_number).
 
-A inversion pair $(i,j)$ is a number pair that satisfied $a_{i} > a_{j}$ and $i < j$.
+An inversion pair $(i,j)$ is, for an array $a$, a number pair that satisfied $a_{i} > a_{j}$ and $i < j$.
 
-<!---
-From Wikipedia:
-> Let $\pi$ be a permutation. If $ i < j $ and $\pi(i)>\pi(j)$, either the pair of places $(i,j)$ or the pair of elements $\bigl (\pi (i),\pi (j)\bigr )$ is called an inversion of $\pi$. 
--->
-<!---原文过于Ambiguous了-->
+The commented statement `ans += mid - p` in previous code implementation is counting the inversion pairs. Specifically, when the algorithm places latter elements to former places, (since smaller elements are placed former), any other element former than the original position and greater than the element forms inversion pairs with this element. The inversion number is the number of elements not yet combined, `mid - p`.
```

Comment:
```suggestion
The commented statement `ans += mid - p` in previous code implementation is counting the inversion pairs. Specifically, when the algorithm places latter elements to former places (since smaller elements are placed former), any other elements former than the original position and greater than the element forms inversion pairs with this element. The inversion number is the number of elements not yet combined, i.e. `mid - p`.
```